{ 'test_motion': { 'left_arm': [ { 'duration': 1.0,
                                   'name': u'arm_up',
                                   'positions': [ -0.0212,
                                                  -0.0108,
                                                  -1.2288,
                                                  0.0049,
                                                  0.0003,
                                                  -0.0001,
                                                  -0.0],
                                   'starttime': 0.0},
                                 { 'duration': 1.0,
                                   'name': u'arm_down',
                                   'positions': [ -0.0424,
                                                  0.0163,
                                                  0.1146,
                                                  0.0108,
                                                  -0.0003,
                                                  0.0,
                                                  0.0],
                                   'starttime': 1.0},
                                 { 'duration': 1.0,
                                   'name': u'arm_up',
                                   'positions': [ -0.0212,
                                                  -0.0108,
                                                  -1.2288,
                                                  0.0049,
                                                  0.0003,
                                                  -0.0001,
                                                  -0.0],
                                   'starttime': 2.0}],
                   'left_leg': [],
                   'neck': [],
                   'right_arm': [],
                   'right_leg': [],
                   'torso': []}}
